pkg/minikube/assets/addons.go

```diff
@@ -621,6 +621,21 @@ var Addons = map[string]*Addon{
 		"GCPAuthWebhook":     "gcr.io",
 		"KubeWebhookCertgen": "registry.k8s.io",
 	}),
+	"volcano": NewAddon([]*BinAsset{
+		MustBinAsset(addons.VolcanoAssets,
+			"volcano/volcano-development.yaml.tmpl",
+			vmpath.GuestAddonsDir,
+			"volcano-deployment.yaml",
+			"0640"),
+	}, false, "volcano", "third-party (volcano)", "hwdef", "", map[string]string{
```

Comment:
please consider adding an Auto Updater Github Action Job that automatically updates the image versions 
simmilar to 
https://github.com/kubernetes/minikube/blob/master/.github/workflows/update-gcp-auth-version.yml